### PR TITLE
Dockerfile Tweaks and Cleanup

### PR DESCRIPTION
**Pull request by egyptiankarim**
Addressing issue #144:

- I corrected the variable expansion issue with `WORKDIR` in the `Dockerfile`.
- Did some light "copy editing".
- Deleted the `entrypoint.sh` script.
- Corrected some of the flags in `run`.

### CODE REVIEW

**Comment by konklone** [comment]
I love deleting code! And it looks good to me, but deferring to @jsf9k as a better Docker reviewer.

**Comment by jsf9k** [comment]
@egyptiankarim, thanks for catching the error and thanks for getting rid of `entrypoint.sh`!

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-# Create an unprivileged user
+# Set up a workspace
 ENV PSHTT_HOME=/home/pshtt
-RUN mkdir ${PSHTT_HOME} 
-RUN groupadd --system pshtt \
-    && useradd --system --comment="pshtt user" --gid="pshtt" pshtt
+RUN mkdir ${PSHTT_HOME}
+
+# Create an unprivileged user
+RUN groupadd --system pshtt
```
Comment (jsf9k) [inline Dockerfile]: @egyptiankarim, do you mind combining these two `RUN` statements like they were originally?  I'm kind of insane about combining related statements where possible to reduce the number of layers in the Docker image.

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+RUN useradd --system --comment="pshtt user" --gid="pshtt" pshtt
 
 # Install pshtt
 COPY . ${PSHTT_HOME}
 RUN chown -R pshtt:pshtt ${PSHTT_HOME}
 RUN pip install --no-cache ${PSHTT_HOME}
 
 # Prepare to run
-WORKDIR PSHTT_HOME
+WORKDIR ${PSHTT_HOME}
```
Comment (jsf9k) [inline Dockerfile]: Great catch!

File: run [hunk 3]
```diff
@@ -3,13 +3,9 @@
...
-
 docker build -t pshtt/cli .
 
 docker run --rm -it \
   --name pshtt \
-  -v $(pwd):/data \
-  `# Change the ownership of the files (**e.g** results) to the user with id 1042 and to the group with id 1042` \
-  -e USER_ID=1042 \
-  -e GROUP_ID=1042 \
+  -v $(pwd):/home/pshtt \
```
Comment (jsf9k) [inline run]: Getting rid of some magic numbers! :+1:

**Comment by konklone** [comment]
@egyptiankarim You should be able to check a box somewhere that lets @jsf9k make changes on the PR directly. I know it lets you do at post-time, not as certain about after posting.

**Comment by egyptiankarim** [comment]
@konklone @jsf9k it's strange because I already have the little "allow edits from maintainers" box checked. Hopefully the latest commit addresses the requested change. Let me know if I need to make any more :)

**Pull request merged**